index.html

```diff
@@ -1,194 +1,198 @@
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hesitate to contact me.</p>
         <button>Get my CV<i class="fa fa-download" aria-hidden="true"></i></button>
         <div class="knowledge">
-        <ul class="lang">
-            <li id="title">Languages</li>
-            <li>JavaScript (ES5/ES6)</li>
-            <li>Ruby</li>
-            <li>HTML</li>
-            <li>CSS</li>
-            <img src="images/ic_languages_mob.svg" alt="testimage" class="hashtag">
-        </ul>
-        <ul class="fram">
-            <li id="title">Frameworks</li>
-            <li>React.js</li>
-            <li>Ruby on rails</li>
-            <li>RSpec</li>
-            <li>Capibara</li>
-            <img src="images/Rectangle 18.png" alt="rectangle18" class="rectangle18">
-            <img src="images/Rectangle 19.png" alt="rectangle19" class="rectangle19">
-        </ul>
-        <ul class="skills">
-            <li id="title">Skills</li>
-            <li>Database Management</li>
-            <li>CLI</li>
-            <li>API Design</li>
-            <li>Version Control</li>
-            <li>Web Development</li>
-            <img src="images/light.png" alt="light-bulb" class="bulb">
-        </ul>
+            <ul class="lang">
+                <li id="title">Languages</li>
+                <li>JavaScript (ES5/ES6)</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <img src="images/ic_languages_mob.svg" alt="testimage" class="hashtag">
+            </ul>
+            <ul class="fram">
+                <li id="title">Frameworks</li>
+                <li>React.js</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capibara</li>
+                <img src="images/Rectangle 18.png" alt="rectangle18" class="rectangle18">
+                <img src="images/Rectangle 19.png" alt="rectangle19" class="rectangle19">
+            </ul>
+            <ul class="skills">
+                <li id="title">Skills</li>
+                <li>Database Management</li>
+                <li>CLI</li>
+                <li>API Design</li>
+                <li>Version Control</li>
+                <li>Web Development</li>
+                <img src="images/light.png" alt="light-bulb" class="bulb">
+            </ul>
 
           <div class="get-contact">
             <img src="images/Shape (1).png" alt="image4" class="image4">
             <img src="images/Shape (2).png" alt="image5" class="image5">
+
+            <div class="contact-page">
             <h1>Get Started</h1>
-            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch</p>
```

Comment:
- [ ] Kindly 😃 as stated by the previous reviewer Great job so far @Team There are just a few changes you need to make. I will suggest that instead of using the h1 tag for the section header, you use one of the other hx tags. ie. (h2, h3, h4 etc) because you already have an h1 tag in the headline section. 💯
- This is essential for a better semantic structure since the h1 tag is the most important heading tag in the HTML structure and it should represent the main heading or title of the page. Using multiple h1 tags can confuse search engines and readers about the main topic of the page. 👍